lsqecc/pauli_rotations/circuit.py

```diff
@@ -75,7 +81,9 @@ def add_single_operator(
 
         self.add_pauli_block(new_rotation, index)
 
-    def apply_transformation(self, start_index: int = 0, remove_y_operators: bool = True) -> None:
+    def apply_transformation(
+        self, start_index: int = 0, remove_y_operators: bool = True
```

Comment:
```suggestion
        self, start_index: int = 0
```
Recommend removing the unused flag

Reply:
Also we would need to check in the package if anything is calling  `apply_transformation` with `remove_y_operators` as well.